app/code/Magento/SendFriend/Helper/Data.php

```diff
@@ -33,6 +33,24 @@ class Data extends \Magento\Framework\App\Helper\AbstractHelper
     const CHECK_IP = 1;
 
     const CHECK_COOKIE = 0;
+    
+    const XML_PATH_SENDER_EMAIL = 'trans_email/ident_support/email';
+
+ /**
+    * Get Sender Email
+    *
+    * @param int $store
+    * @return string
+    */
+
+    public function getSenderEmail($store = null)
```

Comment:
Unfortunately, we are not allowed to add new public methods to the classes marked as @api according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)